src/view.js

```diff
@@ -16,6 +16,7 @@ export default class View {
   }
 
   init() {
+    this.component.node.className += ` shopify-buy-frame shopify-buy-frame--${this.component.typeKey}`;
```

Comment:
that adds  class to the iframe, not the node (which is the div that contains it)